CONTRIBUTING.md

```diff
@@ -84,6 +84,53 @@ This also sets up gulp and Jekyll to watch for file changes to the `/docs`
 and `/src` directories and rebuilds the website accordingly. The command is
 aliased by `npm start`.
 
+#### On a Windows machine:
+Install Node.js and npm. This page is helpful: http://blog.teamtreehouse.com/install-node-js-npm-windows <br>
+Install Ruby and the DevKit from here: http://rubyinstaller.org/downloads/
+```sh
+npm install --global gulp-cli
+```
+
+Then, to start, run the following command to install any new dependencies:
+
+```sh
+npm install
+```
+
+The following examples detail a few tasks you'll encounter as you use gulp:
```

Comment:
Question: Are these tasks folks will definitely encounter, or may encounter? (If the latter, let's say "may encounter.")